tests/estimators/object_detection/test_pytorch_detr.py

```diff
@@ -0,0 +1,300 @@
+# MIT License
+#
```

Comment:
Tried to add to run_tests.sh, but tests failing in setup with `urllib.error.HTTPError: HTTP Error 500: Internal Server Error`, which could be due to needing to pull in the pre-trained weights from `torch.hub`? The PyTorch Faster-RCNN test, which also pulls a pre-trained model from `torch.hub`, is called from a GitHub workflow (`ci-pytorch-object-detectors.yml`) - so that might be a better place to call?